server/Procfile

```diff
@@ -0,0 +1 @@
+web: yarn start
```

Comment:
would definitely need KT for this.
Just wanted to get your attention on filename `Procfile`, should it have been `.procfile` or something?

Reply:
[Procfile](https://devcenter.heroku.com/articles/procfile) is correct.